tabbycat/templates/allocations/DragAndDropStore.js

```diff
@@ -249,20 +249,22 @@ export default new Vuex.Store({
       }
       return false
     },
-    panelClashesForItem: (state) => (id) => {
-      // Note; largely duplicates panelHistoriesForItem
+    panelClashesOrHistoriesForItem: (state) => (id, type) => {
       let panelAdjs = state.debatesOrPanels[id].adjudicators
       let panelAdjIds = Object.values(panelAdjs).map(position => position).flat()
 
-      let panelClashesCombined = { adjudicator: [], team: [], institution: []};
+      let clashesOrHistoriesCombined = { adjudicator: [], team: [], institution: []};
       panelAdjIds.forEach((adjId) => {
-        const clashesForAdj = state.extra.clashes.adjudicators[adjId] ?? { adjudicator: [], team: [], institution: []}
+        const clashesForAdj = state.extra[type].adjudicators[adjId] ?? { adjudicator: [], team: [], institution: []}
         for (const [key, value] of Object.entries(clashesForAdj)) {
-          panelClashesCombined[key].push(...value);
+          clashesOrHistoriesCombined[key].push(...value);
         }
       })
 
-      return panelClashesCombined
+      return clashesOrHistoriesCombined
+    },
+    panelClashesForItem: (state, getters) => (id) => {
+      return getters.panelClashesOrHistoriesForItem(id, 'clashes')
```

Comment:
As you can tell, I'm less a fan of methods merely calling another method, and as used only in `HoverableConflictMixin.vue`, I'd be fine with something like this for `HoverableConflictMixin.vue:19-20`:
```py
clashes = this.panelClashesOrHistoriesForItem(itemId, 'clashes')
histories = this.panelClashesOrHistoriesForItem(itemId, 'histories')
```